ptsprojects/zephyr/sm.py

```diff
@@ -335,6 +335,11 @@ def test_cases(pts):
                   pre_conditions +
                   [TestFunc(btp.gap_set_io_cap, IOCap.no_input_output),],
                   generic_wid_hdl=sm_wid_hdl),
+
```

Comment:
remove the empty line for integrity also on line 333.

Reply:
will do